modules/gdscript/gdscript_functions.cpp

```diff
@@ -267,14 +272,28 @@ void GDScriptFunctions::call(Function p_func, const Variant **p_args, int p_arg_
 			VALIDATE_ARG_NUM(0);
 			r_ret = Math::round((double)*p_args[0]);
 		} break;
-		case MATH_ABS: {
+		case MATH_ABS_INT: {
 			VALIDATE_ARG_COUNT(1);
 			if (p_args[0]->get_type() == Variant::INT) {
 
 				int64_t i = *p_args[0];
 				r_ret = ABS(i);
+				break;
 			} else if (p_args[0]->get_type() == Variant::REAL) {
 
```

Comment:
Fall-through branches should explicitly use `FALLTHROUGH;` macro (assuming that's what all the empty else branches are intended for).

Reply:
I have made the suggested changes in the new commit